src/parser.js

```diff
@@ -147,7 +147,18 @@ export function generateEmbedCode(kvMap,
  * @return {string}
  */
 export function getContent(filePath, originalPath) {
-    return readFileFromPath(filePath);
+    const fileContent = readFileFromPath(filePath);
+    if (fileContent === null) {
+        const urlContent = readFileFromURL(originalPath);
+        if (urlContent === null) {
```

Comment:
This code does that: 

1. Read the file
2. If not found the file, request to any

It may request non-url content.
We should avoid it because it is strange behavior.